provisioning_client/src/prov_device_ll_client.c

```diff
@@ -32,7 +32,7 @@ static const char* JSON_NODE_KEY_NAME = "keyName";
 static const char* JSON_NODE_OPERATION_ID = "operationId";
 static const char* JSON_NODE_ASSIGNED_HUB = "assignedHub";
 static const char* JSON_NODE_TPM_NODE = "tpm";
-static const char* JSON_NODE_TRACKING_ID = "trackingId";
+//static const char* JSON_NODE_TRACKING_ID = "trackingId";
```

Comment:
We either need to keep it (if needed) or remove it. We shouldn't leave commented code..

Reply:
I commented it because I didn't know if it used be needed but isn't needed any more or isn't needed now but will be needed in the future. All I know is it is not needed currently.